run_spammer.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Good practice dictates that scripts starting with shebang should not have the ".sh" extension :)